docs/Worker-Managers.md

```diff
@@ -0,0 +1,71 @@
+# Worker Managers
+
+[Worker Managers](https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker_manager/main.py) 
```

Comment:
cc @pranavjain because we're adding user-facing documentation -- let us know if you think we should add more or if it's clear what is happening here.

Reply:
We can iterate on this.